src/main/java/teammates/ui/controller/FeedbackSubmissionEditSaveAction.java

```diff
@@ -173,7 +173,8 @@ protected ActionResult execute() throws EntityDoesNotExistException {
             removeRespondant();
         }
                
-        if (isSendEmail) {
+        boolean isSendSubmissionEmail = Boolean.valueOf(getRequestParamValue(Const.ParamsNames.SEND_SUBMISSION_EMAIL));
```

Comment:
there is another abstraction that you can use for this: `getRequestParamAsBoolean`.